packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -33,26 +34,76 @@ function getNodeProperty(node, path) {
 }
 
 /**
- * Get a normalized file path. If Embroider prefix is present, strip it out
+ * Function to parse defaults.project and only return info to be used by the plugin
+ * @param {object} project Ember defaults.project
+ * @returns {object} Contains project name and ember-addon path info
+ */
```

Comment:
You don't really need a jsdoc for this, as it's not public. Jsdocs are really for public things that you want consumers to be able to understand. In this case, we just export it to test it, but the only thing public is the babel plugin function.

Reply:
This specific function is public. It is called in `ember-cli-build`. https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata/blob/9172371687e0d7b64edd79d903cfda3f9bfbc94d/packages/test-scenarios/__tests__/scenario-test.js#L23

Reply:
Thanks for pointing that out. Let's document it properly then. Any objects that have specific sub-properties need to be documented using the dot notation.

Example:

```js
/**
 * Function to parse defaults.project and only return info to be used by the plugin
 * @param {object} project - Ember defaults.project.
 * @param {string} project.pkg - The package of the parent project.
 * @param {string} project.pkg.name - The name of the parent project.
 * @returns {object} Contains project name and ember-addon path info
 */
```

The above is just an example, but should give you an idea of how to document sub-properties.